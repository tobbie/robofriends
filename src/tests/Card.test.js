import {shallow} from 'enzyme'
import React from 'react'
import Card from '../components/Card'

describe('card tests',()=> {
    const mockUser = {name : 'Tobi Akinseye', email : 'tobi@example.com'}
    it('expect to render card component',() => {
        expect(shallow(<Card user = {mockUser}/>)).toMatchSnapshot();
        
    });
})




