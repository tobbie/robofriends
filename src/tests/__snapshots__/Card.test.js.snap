// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`card tests expect to render card component 1`] = `ShallowWrapper {}`;
